flowback/chat/views.py

```diff
@@ -32,7 +32,7 @@ def get(self, request, channel_id: int):
         serializer = self.FilterSerializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
 
-        messages = message_list(user=request.user, channel_id=channel_id, filters=serializer.validated_data)
+        messages = message_list(user=request.user, channel_id=channel_id, **serializer.validated_data)
```

Comment:
While this is good, it's better to set filters=None to message_list function aswell as the other selectors present in the file c: